base_business_document_import/models/business_document_import.py

```diff
@@ -274,7 +277,7 @@ def _match_partner_bank(
     def _match_product(self, product_dict, chatter_msg, seller=False):
         """Example:
         product_dict = {
-            'ean13': '5449000054227',
+            'barcode': '5449000054227',
             'code': 'COCA1L',
```

Comment:
In version 8 : Product (String : EAN13 Barcode) field name is ean13 
In version 9 : Product (String: Barcode) field name is barcode